File: V семестр/Паралельне програмування/Кочетов/лабы/lab3/lab3/F1.cs
using System;

namespace lab3
{
    public class F1
    {
    private Vector result;
    public Vector getResult()
    {
        return result;
    }

    private int N;

    public F1(int N)
    {
        this.N = N;
    }

    public void run()
    {
            Console.WriteLine("Task 1 start");
            Vector A = new Vector(N), B = new Vector(N), D = new Vector(N);
            Matrix MA = new Matrix(N), ME = new Matrix(N);
            result = MA.multiply(ME).multiply(A).sum(B).sum(D);
            Console.WriteLine("Task 1 end");
    }
}

}

File: V семестр/Паралельне програмування/Кочетов/лабы/lab7/F1.py
from Matrix import Matrix
from Vector import Vector
class F1:
	def __init__(self, N):
		self.N = N

	def run(self):
		print 'Task 1 start\n'
		MA, ME = [Matrix(self.N) for _ in range(2)]
		A, B, D = [Vector(self.N) for _ in range(3)]
		self.result = MA.multiply(ME).multiply(A).sum(B).sum(D)
		print 'Task 1 end\n'